cime/src/components/data_comps/drof/cime_config/namelist_definition_drof.xml

```diff
@@ -205,12 +258,18 @@
     <group>streams_file</group>
     <desc>Stream data variable name(s).</desc>
     <values>
+      <value>
+        runoff rofl
+      </value>
       <value stream="rof.diatren_ann_rx1">
         runoff rofl
       </value>
       <value stream="rof.diatren_iaf_rx1">
         runoff rofl
       </value>
+      <value stream="rof.diatren_iaf_ais45_rx1">
```

Comment:
@jonbob @matthewhoffman could this be where the AIS45 file is overwriting the standard CORE runoff file?